src/components/text/Subheading.tsx

```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components/native'
+
+export const Subheading = styled.Text`
+  font-size: ${props => props.theme.fonts.defaultFontSize};
+  font-family: ${props => props.theme.fonts.contentFontBold};
+  font-weight: ${props => props.theme.fonts.defaultFontWeight};
+`
+
+export const SubheadingText = styled(Subheading)`
+  color: ${prop => prop.theme.colors.text};
+`
+
+export const SubheadingPrimary = styled(Subheading)`
```

Comment:
I think the naming is a bit misleading, same for other text components, maybe:
```suggestion
export const SubheadingPrimaryColor = styled(Subheading)`
```

Reply:
to keep everything convenient. how should i refactor `content`
`ContentSecondaryColorLight` ?
i spared this out keep it a little bit shorter

Reply:
rejected

Reply:
Why have you rejected this? @klinzo suggested and @Taggotty and me upvoted for it.